### PR TITLE
uutils: 2019-05-03 -> 0.0.2

### PR DESCRIPTION
**Pull request by Atemu**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/misc/uutils-coreutils/default.nix [hunk 1]
```diff
@@ -2,19 +2,25 @@
 , Security
 }:
 
-rustPlatform.buildRustPackage {
-  name = "uutils-coreutils-2019-05-03";
+let
+  pname = "uutils-coreutils";
+
+  version = "0.0.2";
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/uutils-coreutils/default.nix:8]:
```suggestion
  pname = "uutils-coreutils";
  version = "0.0.2";
```

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 109025` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>uutils-coreutils</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 109025` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>uutils-coreutils</li>
</ul>
</details>

File: pkgs/tools/misc/uutils-coreutils/default.nix [hunk 2]
```diff
@@ -2,19 +2,24 @@
...
+
   src = fetchFromGitHub {
     owner = "uutils";
     repo = "coreutils";
-    rev = "036dd812958ace22d973acf7b370f58072049dac";
-    sha256 = "0d9w3iiphhsk7l5l34682wayp90rgq5a3d94l3qdvhcqkfmpg727";
+    rev = version;
+    sha256 = "0rlvwmyk8c597x1krvlyyqhk1m7g2pz2rkwsam8yj3ksb670g53j";
   };
+in
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/uutils-coreutils/default.nix:15]: Please remove that let in.

Comment (Atemu) [reply pkgs/tools/misc/uutils-coreutils/default.nix:15]: Is there a reason for that other than your personal preference?

Comment (SuperSandro2000) [reply pkgs/tools/misc/uutils-coreutils/default.nix:15]: This is how we do it in nixpkgs. Doing let in is usually only done when fetching extra sources.

Comment (SuperSandro2000) [reply pkgs/tools/misc/uutils-coreutils/default.nix:15]: > Is there a reason for that other than your personal preference?

There is no good reason to use a let in here. rec is shorter, cleaner, easier to understand even for non functional programmers and more commonly used which makes it easier to do treewide changes against it. Please change it.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**